sonar-dotnet-shared-library/src/main/java/org/sonar/plugins/dotnet/tests/DotCoverReportParser.java

```diff
@@ -101,6 +104,9 @@ private void collectCoverage(String fileCanonicalPath, String contents) {
         int line = Integer.parseInt(matcher.group(1));
         int hits = Integer.parseInt(matcher.group(2));
         coverage.addHits(fileCanonicalPath, line, hits);
+
+        LOG.trace("DotCover parser: found coverage for line '{}', hits '{}' when analyzing the path '{}'.",
```

Comment:
dotCover website uses lower case letter at the beginning of sentence.
```suggestion
        LOG.trace("dotCover parser: found coverage for line '{}', hits '{}' when analyzing the path '{}'.",
```